packages/extension-starter/src/setup-app.ts

```diff
@@ -80,14 +80,14 @@ class Sample extends ApplicationExtension<Config> {
      */
     // TODO: do we need to pass in additional params like request.path? Get familiar with the SEO API.
     // TODO: how can developers call SCAPI? How can we pass in the api client (with the access token)?
-    getRoutes(): Promise<RouteProps[]> {
-        return Promise.resolve([
+    getRoutes(): RouteProps[] {
+        return [
             {
                 exact: true,
                 path: this.getConfig().path,
                 component: SamplePage
             }
-        ])
+        ]
```

Comment:
refactored this to just return the routes without a promise. I guess by default the `extension-starter` app should implement a synchronous `getRoutes`